lib/jobs/run-sku-graph.js

```diff
@@ -41,7 +41,7 @@ function runSkuGraphJobFactory(
         RunSkuGraphJob.super_.call(this, logger, options, context, taskId);
 
         this.nodeId = this.options.nodeId;
-        assert.isMongoId(this.nodeId);
+        assert.string(this.nodeId);
```

Comment:
Doing this so we can start moving away from mongo-specific string types (in favor of more flexible IDs that work cross-db).